services/api-server/src/simcore_service_api_server/services/solver_job_models_converters.py

```diff
@@ -60,18 +60,19 @@ def create_node_inputs_from_job_inputs(
 
     node_inputs: dict[InputID, InputTypes] = {}
     for name, value in inputs.values.items():
-        assert parse_obj_as(ArgumentTypes, value) == value  # type: ignore  # nosec
+        assert parse_obj_as(ArgumentTypes, value) == value  # nosec # type: ignore
```

Comment:
ignore is missing specs

Reply:
Do you really think it is so relevant when it comes to asserts??